src/atoms/Flex/Flex.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Flex from './Flex';
+import Text from '../Text';
+import View from '../View';
+
+storiesOf('Flex', module)
+  .addParameters({
+    component: Flex,
+  })
+  .add('default', () => (
+    <Flex>
```

Comment:
Flex Component doesn't do anything without props other than cloning the children. You can probably do

```suggestion
    <Flex flex={1} justifyContent="space-evenly">
```

Reply:
added

Reply:
@deorukhkarchaitanya it'll render with default props which is render children as row. So it should be fine